src/fundus/publishers/us/techcrunch.py

```diff
@@ -14,10 +14,42 @@
 
 
 class TechCrunchParser(ParserProxy):
+    class V2(BaseParser):
+        _summary_selector: XPath = CSSSelector("div.entry-content > p#speakable-summary")
+        _paragraph_selector: XPath = CSSSelector("div.entry-content > p:not(#speakable-summary)")
+        _subheadline_selector: CSSSelector = CSSSelector("div.entry-content > h2")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            body: ArticleBody = extract_article_body_with_selector(
+                self.precomputed.doc,
+                subheadline_selector=self._subheadline_selector,
+                summary_selector=self._summary_selector,
+                paragraph_selector=self._paragraph_selector,
+            )
+            return body
+
+        @attribute
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.meta.get("author"))
+
+        @attribute
+        def publishing_date(self) -> Optional[datetime.datetime]:
+            return generic_date_parsing(self.precomputed.ld.bf_search("datePublished"))
+
+        @attribute
+        def title(self) -> Optional[str]:
+            return self.precomputed.meta.get("og:title")
```

Comment:
I would parse this from the JSON, since the og:title tag value has an extra `| TechCrunch` appended to the headline

Reply:
Actually the same problem also occurs with V1. Also, since (almost) only the selectors are changed, I think also a minor parser update (V1_1) would be enough to spare all the duplicate code.

Reply:
The headline referenced in the LD+JSON suffers from encoding problems. I left it as is and removed `| TechCrunch`